%YAML:1.0

use_sim_time: 0
ignore_polarity: 0 # 0: use polarity; 1: ignore polarity
time_surface_mode: 0 # 0: Backward; 1: Forward
decay_ms: 30 # vanilla time surface
median_blur_kernel_size: 0
max_event_queue_len: 100

map_per_msg: 0 # Output TS for each incoming event message
publish_ts: 0 # Publish time surface image

weight_threshold: 0.01
min_valid_pixels: 1000
coeff: 0.1
contrast_threshold: 0.5
filter_size: 1